pages/services/edge-lb/1.4/tutorials/customizing-templates/index.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: layout.pug
+navigationTitle: Customizing Edge-LB templates
+title: Customizing Edge-LB templates
+menuWeight: 10
+excerpt: How to customize Edge-LB load balancing by modifying templates.
+enterprise: true
+---
+Edge-LB uses templates to configure load balancing settings.
+This tutorial illustrates how you can create a custom `haproxy` template for Edge-LB to support Basic authentication.
+
+In this tutoria;, you use settings in the custom `haproxy` template to create at a simple `userlist` that defines the authenticated and authorized users who are allowed access using `frontend` and `backend` configuration settings.
+
+# Before you begin
+* You must have an active and properly-configured DC/OS Enterprise cluster.
+* You must have Edge-LB installed as described in the Edge-LB [installation instructions](/services/edge-lb/1.4/getting-started/installing).
+* You must have the `edgelb` command-line interface (CLI) installed.
+
+# Create a custom template
+
+To create a custom template for Basic authentication:
+
+1. Install Edge-LB as described [Installing Edge-LB](/services/edge-lb/1.4/getting-started/installing).
+
+1. Create an Edge-LB pool as described in [Expose and load balance a service](/services/edge-lb/1.4/getting-started/single-lb).
+
+1. Fetch the default template for the Edge-LB pool and save it to a file names `haproxy.tmpl` by running the following command:
```

Comment:
Even though it is correct, I would like to suggest that instead of fetching the default template, we recommend fetching the template of the given pool. The newly created pool is assigned the default pool anyway, and if somebody is trying to update the existing pool, they would overwrite the existing pool template.

So the command would be:
```
dcos edgelb template show <name of the pool from previous step> > haproxy.tmpl
```

Reply:
changed